sources/commands/Up.ts

```diff
@@ -0,0 +1,49 @@
+import {Command, UsageError} from 'clipanion';
+import semver                from 'semver';
+
+import {BaseCommand}         from './Base';
+
+export class UpCommand extends BaseCommand {
+  static paths = [
+    [`up`],
+  ];
+
+  static usage = Command.Usage({
+    description: `Update the package manager used in the current project`,
+    details: `
+      Retrieve the latest available version for the current major release line of the package manager used in the local project, and update the project to use it.
+
+      Unlike \`corepack use\` this command doesn't accept/need a version range, as it will always select the latest available version from the same major line. Should you need to upgrade to a new major, use an explicit \`corepack use {name}@^{version}\` call.
```

Comment:
Would this work? If so, that's probably what we should document since it's the easiest to remember
```suggestion
      Unlike \`corepack use\` this command doesn't accept/need a version range, as it will always select the latest available version from the same major line. Should you need to upgrade to a new major, use an explicit \`corepack use {name}@latest\` call.
```

Reply:
Yarn doesn't have a `latest` tag available in Corepack, only `stable` and `canary` 🤔

We could hardcode a special case for that I guess.